templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for todo in items %}
+            <div class="flash">{{ todo }}</div>
```

Comment:
You are currently just displaying the whole dictionary as a string, but you could try displaying values more explicitly. So for example you could display just the title and the status, formatted how you want.